crates/symbolicator/src/sources.rs

```diff
@@ -384,11 +384,12 @@ pub enum FileType {
     /// Source bundle
     #[serde(rename = "sourcebundle")]
     SourceBundle,
-    /// PropertyList, mapping a dSYM UUID to a BCSymbolMap UUID for MachO.
-    #[serde(rename = "plist")]
-    PList,
+    /// A file mapping a dSYM UUID to a BCSymbolMap UUID for MachO.
+    ///
+    /// At the time of writing this only supports the XML PropertyList format, but this does
+    /// not need to remain so.
+    BcSymbolMapUuidMap,
     /// BCSymbolMap, de-obfuscates symbol names for MachO.
-    #[serde(rename = "bcsymbolmap")]
```

Comment:
I think you need this serde annotation for parsing? Now your `AsRef<str>` does not match the parser.

Reply:
Oh, I missed that this also has an `AsRef<str>` impl, you're right, they should match.